tiledb/sm/consolidator/fragment_consolidator.cc

```diff
@@ -583,7 +583,9 @@ Status FragmentConsolidator::consolidate_internal(
     vac_uri =
         array_for_reads->array_directory().get_vacuum_uri(*new_fragment_uri);
   } catch (std::exception& e) {
-    std::throw_with_nested(FragmentConsolidatorException(e.what()));
+    std::throw_with_nested(FragmentConsolidatorException(
+        "Internal consolidation failed with exception" +
+        std::string(e.what())));
```

Comment:
```suggestion
    std::throw_with_nested(FragmentConsolidatorException(
        "Internal consolidation failure"));
```

If you're going to call `e.what()`, there's no sense in calling `throw_with_nested`. The whole point is that you don't have to repeat what's already in the nested exception.